Edge/edge on kubernetes (#1399)
SoftwareAG/c8y-docs: content/edge-k8s/manage-edge-bundle/undeploy-edge.md

---
weight: 20
title: Undeploy Edge
layout: redirect
---

> You can undeploy Edge either by:

Reviewer: {{< product-c8y-iot >}} Edge. Also cases below. Please change.

Author: Done.

- Deleting the Edge Custom Resource using the command:
  
   ```shell
   kubectl delete edge c8yedge -n c8yedge
   ```

- Through the [c8yedge-sample.yaml](/files/edge-k8s/c8yedge-sample.yaml) that you used to install {{< product-c8y-iot >}} Edge. 

   For example, you can uninstall by using the command:
   
   ```shell
   kubectl delete -f c8yedge-sample.yaml
   ```
{{< c8y-admon-info >}}

> Substitute the namespace name, which is currently **c8yedge** in the command, with the specific namespace name you've specified in your Edge CR. 

Reviewer 2: ```suggestion
Substitute the namespace name *c8yedge* in the command above with the specific namespace name you have specified in your Edge CR. 
```
